queue/command_queued.go

```diff
@@ -0,0 +1,21 @@
+package queue
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"github.com/micromdm/micromdm/queue/internal/commandqueuedproto"
+)
+
+func MarshalQueuedCommand(udid, uuid string) ([]byte, error) {
```

Comment:
Fair enough. I only did it this way to avoid the proliferation of more little single-use structs.